File: example/javascript/data-map/nodejs/set.js
#!/usr/bin/env node

var map = {
	"apple": "蘋果",
	"banana": "香蕉",
	"cherry": "櫻桃"
};

// set
map["apple"] = "這是蘋果";
map["banana"] = "這是香蕉";
map["cherry"] = "這是櫻桃";

console.log(map["apple"]);
console.log(map["banana"]);
console.log(map["cherry"]);

File: example/lua/data-map/set.lua
#!/usr/bin/env lua

--[[
local map = {
	["apple"] = "蘋果",
	["banana"] = "香蕉",
	["cherry"] = "櫻桃"
}
--]]


local map = {
	apple = "蘋果",
	banana = "香蕉",
	cherry = "櫻桃"
}

-- set
map["apple"] = "這是蘋果。"
map["banana"] = "這是香蕉。"
map["cherry"] = "這是櫻桃。"

print(map["apple"])
print(map["banana"])
print(map["cherry"])
